pages/dkp/kommander/2.2/projects/applications/platform-applications/index.md

```diff
@@ -24,6 +24,6 @@ To use the CLI to enable or disable applications, see [Application Deployment](.
 
 | NAME                           | APP ID                  | Deployed by default |
 | ------------------------------ | ----------------------- | ------------------- |
-| project-grafana-logging-6.16.14 | project-grafana-logging | No                  |
```

Comment:
This version also seems like something we might want to keep

Reply:
yeah I saw a few of these as well. tried to tag them